### ISSUE TITLE
Need "Step by step examples for Windows"

### ISSUE THREAD

**Issue by stakeyourada** [cardano-foundation/incentivized-testnet-stakepool-registry#22]
please add steps using Windows with gitbash and/or Powershell

**Comment by wstrik** [comment]
Hi @stakeyourada,
The Linux script mostly works fine on Windows PowerShell.

**Comment by KtorZ** [comment]
@wstrik If you can share you experience with the PowerShell and maybe, the notable difference such that we can make a `Windows` version of the step-by-step guide, that would be fantastic :tada:

**Comment by mark-stopka** [comment]
@stakeyourada the Linux tutorial works just fine using GitBash, I've done it that way.

**Comment by stakeyourada** [comment]
> @stakeyourada the Linux tutorial works just fine using GitBash, I've done it that way.

yes I used GitBash to get this done also.  However originally they were documenting step-by-step for Linux, and no mention of Windows.  So for Windows users I was suggesting step-by-step instructions also.   In addition while Gitbash did work to create the files many people failed the CI validation step due to EOL conventions.  So some mention of how to configure and handle line-endings for Windows users would be a good thing to add.

https://help.github.com/en/github/using-git/configuring-git-to-handle-line-endings
